proxy/rpc/np_gas_price.py

```diff
@@ -140,3 +168,122 @@ async def get_neon_gas_price(
     async def get_native_token_list(self) -> list[_RpcNativeTokenResp]:
         gas_price = await self._server.get_gas_price()
         return list(_RpcNativeTokenResp.from_raw(token) for token in gas_price.token_dict.values())
+
+    @NeonProxyApi.method(name="eth_maxPriorityFeePerGas")
+    async def get_max_priority_fee_per_gas(self, ctx: HttpRequestCtx) -> HexUIntField:
+        # Fetch the compute units price across last several blocks (as specified in the cfg).
+        num_blocks = self._cfg.priority_fee_num_blocks_to_average
+        assert num_blocks > 0
+        historical_priority_fees: list[list[int]] = await self._db.get_historical_priority_fees(num_blocks)
```

Comment:
Please rename to:
`historical_cu_price_list`

Reply:
done